src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,13 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+
+        System.out.println("Play a lottery!");
+        System.out.println("You get this balls:");
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < 3; i++) {
+            Ball ball = lottery.getRandomBall();
+            System.out.println(ball);
```

Comment:
We don't use ball variable anywhere else so it's creation is not necessary
```suggestion
            System.out.println(lottery.getRandomBall());
```